README.md

```diff
@@ -1,19 +1,19 @@
 # What is this?
 
-An open source project for Gogs documentation.
+This repository is the documentation for the open source project Gogs.
 
-## How to contribute?
+## Contributing
 
 Fork, update and submit a pull request. No matter how big or small, your changes are welcome!
 
-## How does it work?
+## How it works
 
-Gogs Web server checks for updates to this project and updates the main documentation automatically.
+The Gogs web server checks for updates to this project and updates the main documentation automatically.
 
-## Note
+## Formatting
 
-Gogs documentation is written using standard markdown syntax. Please submit your changes using the same syntax.
+Gogs documentation is written using standard [markdown syntax](https://help.github.com/articles/markdown-basics/). Please submit your changes using the same syntax.
 
 ## License
 
-This project is under Apache v2 License. See the [LICENSE](LICENSE) file for the full license text.
+This project is under the Apache version 2.0 license. See [LICENSE](LICENSE) in the repository for the full license text.
```

Comment:
- Use `Apache License, Version 2.0`.
- Please don't change `file` to `in the repository`, it is on purpose.